packages/peregrine/lib/mixins/CategoryList/useCategoryTile.js

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react';
+
+// TODO: get categoryUrlSuffix from graphql storeOptions when it is ready
+const categoryUrlSuffix = '.html';
+const previewImageSize = 480;
+
+export const useCategoryTile = props => {
+    const { item, resourceUrl } = props;
```

Comment:
Big question: If we want to utilize a dynamic `resourceUrl` function that can be overwritten by users of the library we will have to either pass it in as a callback/prop or figure out a mechanism by which to provide this in `peregrine` via drivers.

Thoughts?